compiler/hash-utils/src/state.rs

```diff
@@ -76,19 +76,19 @@ impl<T: Default + Copy + fmt::Debug> LightState<T> {
 /// Heavy state is state which is large in size and cannot be copied cheaply.
 /// Internally uses a [`RefCell`] to store the state.
 #[derive(Debug)]
-pub struct HeavyState<T: fmt::Debug> {
+pub struct HeavyState<T> {
     current: RefCell<T>,
 }
 
```

Comment:
perhaps it should implement debug if `T` implements it?

Reply:
This is what the derive does, the `fmt::Debug` bound was there due to previous debugging and did not get removed.